doc/user_guide/_get-started/03-development-recommendations.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Development Recommendations"
```

Comment:
Add to the https://github.com/intel/rohd-website/blob/main/docs/_data/navigation.yml as well.